catroid/src/androidTest/java/org/catrobat/catroid/uiespresso/content/brick/app/PhiroSeekBarColorBrickTest.java

```diff
@@ -0,0 +1,167 @@
+import org.junit.experimental.categories.Category;
+import org.junit.runner.RunWith;
+
+import static android.support.test.espresso.Espresso.onData;
+import static android.support.test.espresso.Espresso.onView;
+import static android.support.test.espresso.action.ViewActions.click;
+import static android.support.test.espresso.assertion.ViewAssertions.matches;
+import static android.support.test.espresso.matcher.ViewMatchers.withId;
+import static android.support.test.espresso.matcher.ViewMatchers.withText;
+
+import static org.catrobat.catroid.uiespresso.content.brick.utils.BrickDataInteractionWrapper.onBrickAtPosition;
+import static org.hamcrest.Matchers.instanceOf;
+
+@RunWith(AndroidJUnit4.class)
+public class PhiroSeekBarColorBrickTest {
+
+	private static final int MIN_COLOR_VALUE = 0;
+	private static final int MAX_COLOR_VALUE = 255;
+	private static final int INIT_COLOR_VALUE = 128;
+
+	@Rule
+	public BaseActivityInstrumentationRule<ScriptActivity> baseActivityTestRule = new
+			BaseActivityInstrumentationRule<>(ScriptActivity.class, true, false);
+
+	@Before
+	public void setUp() throws Exception {
+		Script script = BrickTestUtils.createProjectAndGetStartScript("PhiroSeekBarColorBrickTest");
+		script.addBrick(new PhiroRGBLightBrick(PhiroRGBLightBrick.Eye.BOTH, INIT_COLOR_VALUE, INIT_COLOR_VALUE, INIT_COLOR_VALUE));
+
+		baseActivityTestRule.launchActivity(null);
+	}
+
+	@Category({Cat.CatrobatLanguage.class, Level.Smoke.class})
+	@Test
+	public void testRedSeekBar() {
+		onBrickAtPosition(0).checkShowsText(R.string.brick_when_started);
+		onBrickAtPosition(1).checkShowsText(R.string.brick_phiro_rgb_led_action);
+
+		onData(instanceOf(PhiroRGBLightBrick.class))
+				.inAdapterView(ScriptListMatchers.isScriptListView())
+				.onChildView(withId(R.id.brick_phiro_rgb_led_action_red_edit_text))
+				.check(matches(withText(Integer.valueOf(INIT_COLOR_VALUE).toString() + " ")));
+		onData(instanceOf(PhiroRGBLightBrick.class))
+				.inAdapterView(ScriptListMatchers.isScriptListView())
+				.onChildView(withId(R.id.brick_phiro_rgb_led_action_red_edit_text))
+				.perform(click());
+
+		onView(withId(R.id.color_rgb_seekbar_red)).perform(swipeLeftSlow());
+		onView(withId(R.id.rgb_red_value)).check(matches(withText(Integer.valueOf(MIN_COLOR_VALUE).toString() + " ")));
+		onView(withId(R.id.color_rgb_seekbar_red)).perform(swipeRightSlow());
+		onView(withId(R.id.rgb_red_value)).check(matches(withText(Integer.valueOf(MAX_COLOR_VALUE).toString() + " ")));
+		onView(withId(R.id.rgb_red_value)).perform(click());
+		onView(withId(R.id.formula_editor_keyboard_ok)).perform(click());
+		onData(instanceOf(PhiroRGBLightBrick.class))
+				.inAdapterView(ScriptListMatchers.isScriptListView())
+				.onChildView(withId(R.id.brick_phiro_rgb_led_action_red_edit_text))
+				.check(matches(withText(Integer.valueOf(MAX_COLOR_VALUE).toString() + " ")));
+	}
+
+	@Category({Cat.CatrobatLanguage.class, Level.Smoke.class})
+	@Test
+	public void testGreenSeekBar() {
```

Comment:
i just saw that this is essentially the same test just with different id's, it would be nice to have all this as parameterized test.